packages/babel-plugin-transform-modules-commonjs/test/fixtures/misc/for-of-in-export/output.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports.bar = exports.foo = void 0;
+let foo;
+exports.bar = exports.foo = foo;
+
+for (let _foo of []) {
+  exports.bar = exports.foo = foo = _foo
```

Comment:
Why is this reassigning `bar`?

Reply:
Because the original input has `export {foo as bar}`.

I don't manually build the export expressions, I use `buildBindingExportAssignmentExpression`.

Reply:
Oh ok :+1: